src/main/resources/application.properties

```diff
@@ -16,38 +19,29 @@ rules.replacement.marker: %IdentifyRulesEventId%
 # This property needs to be enabled in order to test new aggregation rules on an
 # existing or artificially created set of events.
 # WARNING! Do not enable this in a production environment!
-testaggregated.enabled: false
-
-# Logging levels can be set with the flag below for the top packages deactivated by
-# default because Travis will fail due to log file size. It is possible to only
-# activate logging for a certain package under these top packages.
-logging.level.root: OFF
-logging.level.org.springframework.web: ERROR
-logging.level.com.ericsson.ei: ERROR
+test.aggregation.enabled: false
 
 # Details for connection to RabbitMQ
 rabbitmq.host: localhost
 rabbitmq.port: 5672
 rabbitmq.user: myuser
 rabbitmq.password: myuser
-# Check configuration documentation for using jasypt encrypted passswords
 # Valid TLS versions: 'TLSv1.2', 'TLSv1.1', 'TLSv1', 'TLS', 'SSLv3', 'SSLv2', 'SSL'
-rabbitmq.tlsVersion:
+rabbitmq.tls.version:
 rabbitmq.exchange.name: ei-exchange
-rabbitmq.domainId: ei-domain
-rabbitmq.componentName: eiffel-intelligence
-rabbitmq.consumerName: messageConsumer
-rabbitmq.queue.durable:true
-rabbitmq.binding.key:#
-rabbitmq.waitlist.queue.suffix:waitlist
+rabbitmq.domain.id: ei-domain
+rabbitmq.component.name: eiffel-intelligence
+rabbitmq.consumer.name: messageConsumer
+rabbitmq.queue.durable: true
+rabbitmq.binding.key: #
+rabbitmq.waitlist.queue.suffix: waitList
 
 # we reuse the same version as we use in our unit tests
 spring.mongodb.embedded.version: 3.4.1
```

Comment:
Remove this section because it's been duplicated - keep the ones in the bottom only

Reply:
Done